src/components/molecules/EventDisplay/EventDisplay.tsx

```diff
@@ -16,21 +17,26 @@ interface EventDisplayProps {
 }
 
 export const EventDisplay: React.FC<EventDisplayProps> = ({ event, venue }) => {
-  const room = venue?.rooms?.find((room) => room.title === event?.room);
+  const eventRoomTitle = event.room;
+
+  const room = useMemo(
+    () => venue?.rooms?.find((room) => room.title === eventRoomTitle),
+    [eventRoomTitle]
+  );
+
   const buttonText = `${event.room ?? "Enter"} ${venue && `- ${venue.name}`}`;
 
   const isLiveEvent =
     event.start_utc_seconds < getCurrentTimeInUTCSeconds() &&
     event.start_utc_seconds + event.duration_minutes * 60 >
       getCurrentTimeInUTCSeconds();
 
+  const containerClasses = classNames("schedule-event-container", {
+    "schedule-event-container_live": isLiveEvent,
+  });
```

Comment:
Is this `_live` meant to be a 'variant' (aka modifier)? If so, i've been loosely following BEM in my recent styling cleanups, which would make this `--live`:

- http://getbem.com/naming/